src/gfx/qt.cpp

```diff
@@ -378,7 +378,8 @@ int GfxProcQT::processEXIF(QByteArray *data, int itemlen){
 }
 /************* END OF EXIF STUFF **************/
 
-#if defined(HAVE_PDFIUM) && defined(_WIN32)
+// turn on PDFIUM_DELAY_LOAD_DLL if you are building XP-compatible apps and using pdfium.dll (which won't load on XP)
+#if PDFIUM_DELAY_LOAD_DLL
```

Comment:
Shouldn't we have this defined in `sdk.pri` for the moment as default when win32 & USE_PDFIUM ?? and only remove it after ceasing XP support?

Reply:
using the vcpkg/cmake system, we can build pdfium statically, and I think that will work on XP (I have not tried running it, but compiling and linking for XP works).  In case I'm wrong, we can easily turn the delay load back on.